ArcadiaParties-SPA/src/app/header/header.component.html

```diff
@@ -1,4 +1,3 @@
-<div>{{ title$ | async }}</div>
+<a class="cursor" [routerLink]="['/home']">{{ title$ | async }}</a>
```

Comment:
Should have static text, because won't be changed.

Reply:
I've changed it to static text